com.unity.render-pipelines.universal/Runtime/ScriptableRendererFeature.cs

```diff
@@ -48,5 +48,18 @@ public void SetActive(bool active)
         {
             m_Active = active;
         }
+
+        /// <summary>
+        /// Disposable pattern implementation.
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
Is this needed when we don't have a finalizer? Same goes for the `bool disposing` parameter actually.